packages/starter/example/pages/index.tsx

```diff
@@ -26,107 +27,126 @@ import { useAutoConnect } from '../components/AutoConnectProvider';
-                            </TableRow>
-                            <TableRow>
-                                <TableCell>Multi Button</TableCell>
-                                <TableCell>
-                                    <MaterialUIWalletMultiButton />
-                                </TableCell>
-                                <TableCell>
-                                    <AntDesignWalletMultiButton />
-                                </TableCell>
-                                <TableCell>
-                                    <ReactUIWalletMultiButton />
-                                </TableCell>
-                                <TableCell></TableCell>
-                            </TableRow>
-                            <TableRow>
-                                <TableCell></TableCell>
-                                <TableCell>
-                                    <Tooltip title="Only runs if the wallet is ready to connect" placement="left">
-                                        <FormControlLabel
-                                            control={
-                                                <Switch
-                                                    name="autoConnect"
-                                                    color="secondary"
-                                                    checked={autoConnect}
-                                                    onChange={(event, checked) => setAutoConnect(checked)}
-                                                />
-                                            }
-                                            label="AutoConnect"
-                                        />
-                                    </Tooltip>
-                                </TableCell>
-                                <TableCell>
-                                    <RequestAirdrop />
-                                </TableCell>
-                                <TableCell>
-                                    <SendTransaction />
-                                </TableCell>
-                                <TableCell>
-                                    <SignMessage />
-                                </TableCell>
-                            </TableRow>
-                        </TableBody>
-                    </Table>
-                </ReactUIWalletModalProvider>
+                <AntDesignConnectionModalProvider
+                    connections={[
+                        {
+                            endpoint: clusterApiUrl(WalletAdapterNetwork.Mainnet),
+                            name: 'Mainnet',
+                        },
+                        {
+                            endpoint: clusterApiUrl(WalletAdapterNetwork.Devnet),
+                            name: 'Devnet',
+                        },
+                        {
+                            endpoint: clusterApiUrl(WalletAdapterNetwork.Testnet),
+                            name: 'Testnet',
```

Comment:
Names like this should be an enum or some other typesafe structure so that child components are sure the input the correct values. The `connections` array could be part of the `ConnectionProvider` since it's not dependent upon UI, and the `connection` value can be an item in that array.